src/ontology/foodon-edit.owl

```diff
@@ -57823,12 +57838,17 @@ SubClassOf(obo:FOODON_03460172 obo:FOODON_00002454)
 # Class: obo:FOODON_03460173 (food (salted))
 
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03460173 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03460173 "requires curation"@en)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03460173 "A food product which is output of a salting process where salt is dissolved or absorbed into the food as an ingredient or flavourant."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03460173 "http://langual.org")
+AnnotationAssertion(dc:contributor obo:FOODON_03460173 "https://orcid.org/0000-0001-5193-0062 | https://orcid.org/0000-0001-8008-8249"@en)
```

Comment:
Perhaps you want to address this now: each dc:contributor gets its own entry, we don't pipe them into one entry.  As well when editing them the ORCID URL should go in the "IRI Editor" tab of the popup.

Reply:
Absolutely, sorry about this - must've copied and pasted from the ROBOT. Thanks for catching that! Have amended in later commit.